.travis.yml

```diff
@@ -13,7 +13,8 @@ install:
 - ./install_dependencies.sh
 
 script:
-- coverage run --source="./matchmaker","./seqr","./reference_data" --omit="*/migrations/*","*/apps.py" manage.py test -p '*_tests.py' -v 2 reference_data seqr matchmaker
+- TERRA_API_ROOT_URL=https://localhost/ coverage run --source="./matchmaker","./seqr","./reference_data" --omit="*/migrations/*","*/apps.py" manage.py test -p '*_tests.py' -v 2 reference_data seqr matchmaker
+- coverage run -a --source="./matchmaker","./seqr","./reference_data" --omit="*/migrations/*","*/apps.py" manage.py test -p '*_tests.py' -v 2 reference_data seqr matchmaker
```

Comment:
I finally don't use `test detection` in the settings because it makes the tests always with the `TERRA_API_ROOT_URL` setting. We need to run the tests twice, one with `TERRA_API_ROOT_URL` and the other without it. The option `-a` of `coverage run` command can combine the results of the two runs.
I'm not sure whether it works with `Travis` or not by adding `TERRA_API_ROOT_URL=https://localhost/` in front of the test command. But, just try it first.

Reply:
If there is an option for `manage.py test` which can be detected by the `settings.py` then it will be perfect.

Reply:
As I mentioned yesterday, we used to do this in seqr code. Here is an example: https://github.com/broadinstitute/seqr/blob/19a1266e267ed2c5197c5f976d58114829865b08/settings.py#L423

Reply:
The problem with the detection in this way is that it always sets `TERRA_API_ROOT_URL` when it is testing. We need tests for both `TERRA_API_ROOT_URL` set and un-set.